Docker/docker-compose-test.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.9'
+services:
+  mysql:
+    build:
+      context: ./mysql
+    environment:
+      MYSQL_USER: efes
+      MYSQL_PASSWORD: 123
+      MYSQL_DATABASE: efes
+  rabbitmq:
+    build:
+      context: ./rabbitmq
+    environment:
+      RABBITMQ_DEFAULT_USER: efes
+      RABBITMQ_DEFAULT_PASS: 123
+      RABBITMQ_NODENAME: efes@localhost
+    logging:
+      driver: none
+  tracker:
```

Comment:
There was no tracker service in old setup. Why do we require it now?

Reply:
Good catch! I blindly replaced all localhosts with container names. Where I shouldn't have done for tracker url in config. Will fix it.